src/main.py

```diff
@@ -56,7 +58,11 @@ def setup(self):
             self.arena,
         )
 
-        game_mode = StockGameMode(self.player_manager.players)
+        if GAME_MODE == "stock":
```

Comment:
Do we have a convention for this kind of magic string?  Typically these would be constants, kind of like constants.py.  Maybe in a file called options.py or something?